docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,32 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="control"></a>
```

Comment:
Please check 'name'. It seems that `control` is duplicated.

Reply:
Thank you for checking this. I've fixed it.